README.md

```diff
@@ -1,3 +1,373 @@
+| `on_warning_callback` | `Callable[[BaseException, Optional[float]], None]`, optional | `None` | Callback function triggered when a warning occurs. |
+
+__Examples__
+
+Create a new run:
+
+```python
+from neptune_scale import Run
+
+with Run(
+    project="team-alpha/project-x",
+    api_token="h0dHBzOi8aHR0cHM6...Y2MifQ==",
+    family="aquarium",
+    run_id="likable-barracuda",
+) as run:
+    ...
+```
+
+> [!TIP]
+> Find your API token in your user menu, in the bottom-left corner of the Neptune app.
+>
+> Or, to use shared API tokens for multiple users or non-human accounts, create a service account in your workspace settings.
+
+Create a forked run and mark it as an experiment:
+
+```python
+with Run(
+    family="aquarium",
+    run_id="adventurous-barracuda",
+    experiment_name="swim-further",
+    fork_run_id="likable-barracuda",
+    fork_step=102,
+) as run:
+    ...
+```
+
+Continue a run:
+
+```python
+with Run(
+    family="aquarium",
+    run_id="likable-barracuda",  # a Neptune run with this ID already exists
+    resume=True,
+) as run:
+    ...
+```
+
+### `close()`
+
+The regular way to end a run. Waits for all locally queued data to be processed by Neptune (see [`wait_for_processing()`](#wait_for_processing)) and closes the run.
+
+This is a blocking operation. Call the function at the end of your script, after your model training is completed.
+
+__Examples__
+
+```python
+from neptune_scale import Run
+
+run = Run(...)
+
+# logging and training code
+
+run.close()
+```
+
+If using a context manager, Neptune automatically closes the run upon exiting the context:
+
+```python
+with Run(...) as run:
+    ...
+
+# run is closed at the end of the context
+```
+
+### `log_configs()`
+
+Logs the specified metadata to a Neptune run.
+
+You can log configurations or other single values. Pass the metadata as a dictionary `{key: value}` with
+
+- `key`: path to where the metadata should be stored in the run.
+- `value`: the piece of metadata to log.
+
+For example, `{"parameters/learning_rate": 0.001}`. In the field path, each forward slash `/` nests the field under a namespace. Use namespaces to structure the metadata into meaningful categories.
+
+__Parameters__
+
+| Name          | Type                                               | Default | Description                                                               |
+|---------------|----------------------------------------------------|---------|---------------------------------------------------------------------------|
+| `timestamp`   | `datetime`, optional                               | `None`  | Time of logging the metadata. |
+| `data`      | `Dict[str, Union[float, bool, int, str, datetime, list, set]]`, optional  | `None` | Dictionary of configs or other values to log. Available types: float, integer, Boolean, string, and datetime. |
```

Comment:
We should either remove the `list` and `set` or mention that StringSets are available:
```python
run.log_configs(
        data={"learning_rate": 0.001, "batch_size": 64, "test": ["1", "2", "3"]}
    )
```